### PR TITLE
support yaml output

### PR DESCRIPTION
**Pull request by danielerez**
Added support for multiple output printers:
- yaml
- json (default)

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -9,17 +9,21 @@ import (
 	"github.com/matt-simons/ss/pkg"
 	hivev1 "github.com/openshift/hive/pkg/apis/hive/v1"
 	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
```
Comment (ahadas) [inline cmd/root.go]: @danielerez  can we use "sigs.k8s.io/yaml" as it is already used in pkg/ss.go?

File: cmd/root.go [hunk 2]
```diff
@@ -70,11 +78,33 @@ var viewCmd = &cobra.Command{
...
+		b, err = json.MarshalIndent(&ss, "", "    ")
+	}
+	if err != nil {
+		log.Fatalf("error: %v", err)
+	}
+	return string(b)
+}
+
+// Contains tells whether array a contains member x.
+func Contains(a []string, x string) bool {
```
Comment (ahadas) [inline cmd/root.go]: Contains -> contains

File: go.mod [hunk 3]
```diff
@@ -10,6 +10,7 @@ require (
 	github.com/openshift/hive v0.0.0-20200213212254-ed7dfcb60d71
 	github.com/spf13/cobra v0.0.5
 	golang.org/x/net v0.0.0-20200202094626-16171245cfb2 // indirect
+	gopkg.in/yaml.v2 v2.2.8
```
Comment (ahadas) [inline go.mod]: and this won't be needed

File: cmd/root.go (hunk 2 above)
Comment (danielerez) [reply cmd/root.go]: it should probably be in some public helper, but will change for now.

File: go.mod (hunk 3 above)
Comment (danielerez) [reply go.mod]: done

File: cmd/root.go (hunk 1 above)
Comment (danielerez) [reply cmd/root.go]: done

**Review by ahadas — APPROVED** [review]

File: cmd/root.go [hunk 4]
```diff
@@ -70,11 +78,33 @@ var viewCmd = &cobra.Command{
 			}
 			var ss hivev1.SelectorSyncSet
 			ss = pkg.CreateSelectorSyncSet(args[0], selector, resources, patches)
-			j, err := json.MarshalIndent(&ss, "", "    ")
-			if err != nil {
-				log.Fatalf("error: %v", err)
-			}
-			fmt.Printf("%s\n\n", string(j))
+			fmt.Printf("%s\n\n", GetOutputStr(ss))
```
Comment (ahadas) [inline cmd/root.go]: need to change here to getOutputStr

**Pull request closed without merging**